src/main/java/io/jenkins/plugins/zscaler/models/Region.java

```diff
@@ -1,16 +1,18 @@
 package io.jenkins.plugins.zscaler.models;
 
 public enum Region {
-  US("US", "https://api.zcpcloud.net", "https://auth.us.zcpcloud.net");
+  US("US", "https://api.zcpcloud.net", "https://auth.us.zcpcloud.net", "https://app.zcpcloud.net/iacplugin");
```

Comment:
The URL is different now and hosted with different subdomain. Please correct it.